src/Backoffice/Airline/App/Request/StoreAirlineRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Airline\Domain\DataTransferModels\AirlineDto;
+
+class StoreAirlineRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public const DESCRIPTION = 'description';
+
+    public const CITIES = 'cities';
+
+    public function rules(): array
+    {
+        return [
+            self::NAME => ['required', 'string', 'max:255', 'unique:airlines'],
+            self::DESCRIPTION => ['required', 'string'],
+            self::CITIES => ['required', 'array'],
+            self::CITIES . '.*' => ['exists:cities,id'],
```

Comment:
Can't do this like this?

```suggestion
            self::CITIES => ['required', 'array', 'exists:cities,id'],
```